packages/api-explorer/src/form-components/SchemaField.jsx

```diff
@@ -9,6 +9,25 @@ function getDefaultNumFormat(type) {
   return '';
 }
 
+function doesFormatExist(type, format) {
+  const availableFormats = [
```

Comment:
You can dynamically get a list of these from `props.registry.widgets` inside of this function: https://github.com/readmeio/api-explorer/pull/202/files#diff-ca60764ec4fc812c3c70557738a148f0R49

This is the result of a merge of the widgets that we pass in + the defaults. This will save us from having to update this list in future if we add more. Alternatively, we could export this object from here https://github.com/readmeio/api-explorer/blob/master/packages/api-explorer/src/Params.jsx#L50-L61 and utilize it in here.